src/components/extension-card/ExtensionCard.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ReactNode, useState } from 'react';
+import styled, { css } from 'styled-components';
+import { Button } from '..';
+import { SFlex } from '../../base-components/flex/styles/SFlex';
+import { IButtonType } from '../button/Button';
+import { FlatActionButton } from '../flat-action-button/FlatActionButton';
+import { SExtensionCard } from './styles/SExtensionCard';
+import { SExtensionCardDescription } from './styles/SExtensionCardDescription';
+import { SExtensionCardTitle } from './styles/SExtensionCardTitle';
+
+interface IExtensionCard {
+  titleIcon?: ReactNode;
+  title?: string | undefined;
+  description?: string | undefined;
+  flatButtonIcon?: ReactNode;
+  flatButtonText?: string | undefined;
```

Comment:
@steiningermona the interface is missing a callback to handle the button action. There should be 2: `onAdd` and `onRemove`